packages/components/text/src/Text.tsx

```diff
@@ -79,6 +83,9 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
     [onPress, onAccessibilityTap],
   );
 
+  // TODO(#2268): Remove once RN Core properly supports Dynamic Type scaling
+  const dynamicTypeVariant = Platform.OS === 'ios' ? tokens.dynamicTypeRamp : undefined;
+
```

Comment:
Any reason this has to be here and not right before line 117? If not, could we add this to be inside the `Platform.OS === 'ios'` check I suggested earlier?

Reply:
I think this is cleaner because we also check `dynamicTypeVariant` in the continuation where we define `mergedProps`. This also ensures that `dynamicTypeVariant` is only used when it's actually relevant.